src/utils/AppRouteProvider.jsx

```diff
@@ -43,7 +45,19 @@ const AppRouteProvider = (props) => {
   const router = useRouter();
   const dispatch = useDispatch();
 
+  const pathLookup = useRef(Object.entries(PATH_STUBS));
```

Comment:
why do we need that `useRef` ?

Reply:
So taht it's not calculated again and again on every rerender.

Reply:
`PATH_STUBS` is a small object, overhead of importing `useRef` might not be worth it, so removed.